### PR TITLE
Ns search

### PR DESCRIPTION
**Pull request by natibekele**
added capability to search
- add data-* attributes to card fragments
- small css fixes

---
This change is [Reviewable](https://reviewable.io/reviews/bible-translation-tools/fetcher/34)

### CODE REVIEW

**Comment by AnonymousWalker** [comment]
@natibekele Have you tried the phone & tablet mode? It doesn't look right on my side

**Comment by 16dprice** [comment]
___
*[src/main/resources/css/common.css, line 75 at r3](https://reviewable.io/reviews/bible-translation-tools/fetcher/34#-MDpKms_BRbA5_BfHXXg:-MDpKms_BRbA5_BfHXXh:b-h957nj) ([raw file](https://github.com/bible-translation-tools/fetcher/blob/64ac48c7e0cb1f673650af148c7019b170ed10c6/src/main/resources/css/common.css#L75)):*
> ```CSS
>     width: 100%;
>     height: 100%;
> }
> ```



I think this CSS is causing this weird stuff to happen with the navbar.

File: src/main/resources/css/common.css [hunk 1]
```diff
@@ -61,14 +61,26 @@ span {
...
+    height: 100%;
+}
 .navbar {
     display: flex;
     align-items: center;
     justify-content: space-evenly;
     width: 100%;
-    height: 56px;
+    max-width: 900px;
+    height: 100%;
```
Comment (AnonymousWalker) [inline src/main/resources/css/common.css:82]: I think this one causes the nav bar to display incorrectly on mobile version. When I remove height: 100%, it looks like this (with the offset on top)  which can be rendered accordingly, too
Capture

Comment (natibekele) [reply src/main/resources/css/common.css]: I forgot to push up that change. look at the latest commit, it should be working

**Comment by AnonymousWalker** [comment]
___
*[src/main/resources/css/chapters.css, line 39 at r8](https://reviewable.io/reviews/bible-translation-tools/fetcher/34#-ME2sOgsE_Dp2t0H7OI7:-ME2xbfe-ch43MKIPTMR:b-886gsx) ([raw file](https://github.com/bible-translation-tools/fetcher/blob/b1864cde41021b68b596fcb6f1f6e6fbe0c3af42/src/main/resources/css/chapters.css#L39)):*
<details><summary><i>Previously, jsarabia (Joe) wrote…</i></summary><blockquote>

indent
</blockquote></details>

Fixed

**Comment by AnonymousWalker** [comment]
___
*[src/main/resources/templates/fragments/filter.html, line 18 at r8](https://reviewable.io/reviews/bible-translation-tools/fetcher/34#-ME2slx98crB3VgLc5S6:-ME2xeFR63IDSZ9IWDxy:b-896fja) ([raw file](https://github.com/bible-translation-tools/fetcher/blob/b1864cde41021b68b596fcb6f1f6e6fbe0c3af42/src/main/resources/templates/fragments/filter.html#L18)):*
<details><summary><i>Previously, jsarabia (Joe) wrote…</i></summary><blockquote>

some indentation here contains an extra space
</blockquote></details>

Done!

**Pull request merged**